src/styles/blocks/catalog.scss

```diff
@@ -17,9 +17,16 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    cursor: pointer;
+    transition: color 300ms linear, transform 300ms linear;
```

Comment:
add the repeatable values to css variables. you have written `300ms` several times